cmake/modules/FindWaylandpp.cmake

```diff
@@ -22,6 +24,13 @@ if(PKG_CONFIG_FOUND)
       pkg_get_variable(PC_WAYLAND_PROTOCOLS_DIR wayland-protocols pkgdatadir)
     endif()
   endif()
+
+  if(PC_WAYLANDPP_FOUND AND PC_WAYLANDPP_wayland-client++_VERSION VERSION_LESS WAYLANDPP_MINIMUM_VERSION)
+    message(FATAL_ERROR "At least version ${WAYLANDPP_MINIMUM_VERSION} of waylandpp is required, but you only have version ${PC_WAYLANDPP_wayland-client++_VERSION}")
```

Comment:
should be handled by `find_package_handle_standard_args` (see FindBluray.cmake for an example)

Reply:
If you use `find_package_handle_standard_args`, that means that the FindPackage call in the actual CMake file must request a `VERSION`, which is not the case.

Reply:
Only `set(Waylandpp_FIND_VERSION 0.1.5)` would be needed before `find_package_handle_standard_args` call and version var has to added to that.

Reply:
Not sure it's intended to be used like that, but yeah that should work :-)

Reply:
We use it like that for other modules already.

Reply:
done